mainnet-dryrun-2/gentx/gentx-fb8947a283d330580353b52e740ca3a9f2e6b529.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Spectrum","identity":"715E454ACF0C537C","website":"https://spectrumstaking.net","security_contact":"info@spectrumstaking.net","details":"Reliable and secure blockchain validation services by experienced SRE team."},"commission":{"rate":"0.050000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"10","delegator_address":"archway1c37x9qfka8ms3pmqkxl39vd8gks6d0dalpuw0g","validator_address":"archwayvaloper1c37x9qfka8ms3pmqkxl39vd8gks6d0dal94vzp","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"7A0HZlPzGOSgYCePVc0AIkwrimkI71TNoazgEOLUGSc="},"value":{"denom":"aarch","amount":"8000000000000000000"}}],"memo":"fb8947a283d330580353b52e740ca3a9f2e6b529@192.168.1.105:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ay//fBEzgwK+emOP4ht6YbGqJnhKELqsjR262Q2sY0v4"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["ioAwPoou+yki6uPsJH7/WMEuqaN8raWkX9TB0uQnspgCKVDA0coFLaEPdd7IkKLt+aPvib0FQxavlmJVNwqMXg=="]}
```

Comment:
Gentx fee is empty
Check docs https://docs.archway.io/validators/becoming-a-validator/running-a-validator-node#create-your-validator

Reply:
Fixed.